Documentation/markdown/SupportedIFCrepresentations.md

```diff
@@ -24,7 +24,92 @@ Meaning of columns:
 | Unit test | does OIP have unit tests for this entity; denoted by symbol, see [legend above](#legend) |
 | Issue | Link to corresponding open issue(s) |
 
-| Entity                | Covered              | Converter                  | Unit test             | Issue |
-|-----------------------|----------------------|----------------------------|-----------------------|-------|
-| `IfcCartesianPoint`   | :heavy_check_mark:   | `PlacementConverter`       | :heavy_check_mark:    | -     |
-| `IfcPointOnCurve`     | :x:                  | -                          | -                     | [#359](https://www.github.com/tumcms/Open-Infra-Platform/issues/359) |
+| Entity                                | Covered              | Converter                  | Unit test             | Issue |
+|---------------------------------------|----------------------|----------------------------|-----------------------|-------|
+| `IfcAdvancedBrep`                     | :clock9:             | `SolidModelConverter`      | -                     | -     |
+| `IfcAdvancedBrepWithVoids`            | :clock9:             | `SolidModelConverter`      | -                     | -     |
+| `IfcAlignmentCurve`                   | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcAnnotationFillArea`               | :x:                  | -                          | -                     | -     |
+| `IfcAxis1Placement`                   | :clock9:             | `PlacementConverter`       | -                     | -     |
+| `IfcAxis2Placement2D`                 | :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcAxis2Placement3D`                 | :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcAxis2PlacementLinear`             | :x:                  | `PlacementConverter`       | -                     | -     |
+| `IfcBlock`                            | :heavy_check_mark:   | `SolidModelConverter`      | -                     | -     |
+| `IfcBlossCurve`                       | :x:                  | `CurveConverter`           | -                     | -     |
+| `IfcBooleanClippingResult`            | :clock9:             | `SolidModelConverter`      | -                     | -     |
```

Comment:
I believe this one should be supported already as well - perhaps not in a separate function, but as part of CSG handling. Double-check, please.

Reply:
![image](https://user-images.githubusercontent.com/76201952/110937715-11cab480-8333-11eb-838f-3e217afb28f1.png)
The code is commented, so I habe doubts

Reply:
Understandably. 

It is supported, `IfcBooleanClippingResult` only introduces some conditions on the `IfcBooleanResult`, it is the same data and requires the same handling.

Reply:
Resolved 31a5bd4